src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1702,6 +1702,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         ERR_ValueTupleTypeRefResolutionError = 37267
 
+        ERR_TupleElementNamesAttributeMissing = 37268
+        ERR_ExplicitTupleElementNames = 37269
```

Comment:
Consider adding `Attribute` suffix: `ERR_ExplicitTupleElementNamesAttribute`. Same comment for C#.